_posts/2020-09-17-using-tettra.md

```diff
@@ -0,0 +1,82 @@
+
+If a new category is created or an existing one is removed, the Knowledge Base Structure page also needs to be updated to reflect it.
+
+## Different Types of Content
+
+When talking about text, we can add content to Tettra directly through a page or a link to an external Google Doc or Sheet. However, when it comes to content in the form of videos, PDF documents, image files or audio, we need a place to store those files.
+
+We use Google Drive for that, so the content is organized and available to the whole team. However, links to those files exist inside a Tettra page so everything is accessible from within the knowledge base. This way, we're able to keep the content centralized.
+
+## Slack Integration
+
+Tettra integrates nicely with [Slack](https://slack.com), our tool of choice for internal communication. Since our team is in Slack a lot, we decided integrating our knowledge base with it was pivotal.
+
+The Slack integration allows us to notify a Slack channel every time a new page is published or an existing page is updated. We have specific categories linked to specific channels and when a page or changes to a page are published, we just need to click a button to notify the team on Slack.
+
+Through the integration we can also easily search for content. Tettra offers a variety of Slack commands that allow us to search for content, add a new page or create a new suggestion.
+
+Finally, the integration also adds the Tettra Wiki bot, which notifies a user in Slack when they are tagged in a Tettra page or suggestion.
+
+## Custom Integration
+
+When we thought of the knowledge base, it was important to us to connect it to our blog posts, as it contains a great deal of information on our procedures and how we use the tools we choose to use.
+
+In order to keep those synced, we created a custom integration via Tettra's API. This way, all the information available in our blog is also accessible from the knowledge base.
+
+## Keeping the Knowledge Base Up to Date
+
+Documentation becomes obsolete, policies change and so on. Adding content is a good way to make sure all the information is safely stored, but removing obsolete content and updating existing pages is also important to keep the knowledge base clean and relevant.
+
+In order to ensure our content is always up to date, we use Tettra's inbuilt feature of notifying the admin every time a page has gone three months without an update. The admin gets a notification and can check the page in question and update it if needed or tag the best team member to review and update.
+
+Having a notification in Slack is a great way to ensure the admin knows when to review pages and which pages to review. And having the clock running for each page individually ensures we don't have to dedicate a lot of time to review the entire knowledge base ever so often.
+
+## Conclusion
```

Comment:
There is a lot of "easy" in the last paragraph...consider:
It integrates nicely with the tools we use, has an easy to use interface and allows us to  keep the content updated and accessible.